src/main/java/org/jabref/logic/pdf/PdfAnnotationImporter.java

```diff
@@ -71,6 +67,10 @@
     }
 
     private boolean isSupportedAnnotationType(PDAnnotation annotation) {
+        if (annotation.getSubtype().equals("Link") || annotation.getSubtype().equals("Widget")) {
```

Comment:
I don't know how likely it is, but just to be sure you should change it to "Link".equals...  otherwise there might be a potenial NPE